src/main/config/secor.common.properties

```diff
@@ -398,3 +398,9 @@ parquet.enable.dictionary=true
 # Enable or disable validation for Parquet writers. Validates records written against the schema. Defaults to false in
 # Parquet 1.9.
 parquet.validation=false
+
+# User can configure ORC schema for each Kafka topic. Common schema is also possible. This property is mandatory
+# if DefaultORCSchemaProvider is used. ORC schema for all the topics should be defined like this:
+secor.orc.message.schema.*=struct<a:int\,b:int\,c:struct<d:int\,e:string>\,f:array<string>\,g:int>
```

Comment:
Yes Henry. User can specify ORC schema per topic and fallback schema is also there. It is also possible to have custom implementation of ORC schema provider.